### PR TITLE
[6.1.z] DockerBrowser - name container by test_case for better readibility

### PR DESCRIPTION
**Pull request by rplevka**
cherry-picking ccd00d3d12cd1a7f4900ab32030e1f48844b53ab

### CODE REVIEW

File: robottelo/ui/browser.py [hunk 1]
```diff
@@ -173,6 +176,8 @@ def _create_container(self):
             host_config=self._client.create_host_config(
                 publish_all_ports=True),
             image='selenium/standalone-firefox',
+            name=self._name.split('.', 4)[-1] + '_{0}'.format(
```
Comment (elyezer) [inline robottelo/ui/browser.py]: If name is None then this will raise an exception. Better to check if the name is set before calling split.

**Comment by rplevka** [comment]
@elyezer, how about now?

File: robottelo/ui/browser.py [hunk 2]
```diff
@@ -1,4 +1,6 @@
 """Tools to help getting a browser instance to run UI tests."""
+from fauxfactory import gen_string
```
Comment (elyezer) [inline robottelo/ui/browser.py]: Move the `from .. import ..` after the `import` lines

Comment (rplevka) [inline robottelo/ui/browser.py]: addressed

**Comment by elyezer** [comment]
Travis is not happy because ``/robottelo/ui/browser.py:2:1: F401 'logging' imported but unused``

**Comment by renzon** [comment]
@rplevka code is ok, just merge it after removing unused import Travis is complaining about.

**Comment by rplevka** [comment]
@renzon sorry for the delay. addressed.

**Pull request merged**